app/components/home/HomeCard.vue

```diff
@@ -138,13 +135,7 @@ const columns: TableColumn<Score>[] = [
         <div class="absolute left-0 right-0 flex justify-center -bottom-4 gap-x-4">
           <UButton
             class="relative"
-            :class="[
-              canUnlockNuxterBadge ? 'primary-button' : 'bg-neutral-900',
-              { 'cursor-auto hover:bg-neutral-950': !canUnlockNuxterBadge },
-              { 'primary-button-discord': !linked.discord && canUnlockNuxterBadge },
-              { 'cursor-auto hover:bg-neutral-950 text-primary-400': linked.discord && canUnlockNuxterBadge },
-            ]"
```

Comment:
don't we want to have a different colour if it's possible to unlock the discord badge?